cmd/conduit/main.go

```diff
@@ -14,6 +15,7 @@ import (
 	"github.com/algorand/indexer/cmd/conduit/internal/list"
 	"github.com/algorand/indexer/conduit"
 	"github.com/algorand/indexer/conduit/pipeline"
+
```

Comment:
the lint error is related to the import order in this file. It is fixed on develop.

Reply:
Thanks, the error message was so helpful.

Reply:
Actually I merged develop and it doesn't change anything about the import order as far as I can tell, and the problem persists.